src/platform/unix.rs

```diff
@@ -345,9 +345,31 @@ pub trait WindowBuilderExtUnix {
     #[cfg(feature = "x11")]
     fn with_gtk_theme_variant(self, variant: String) -> Self;
     /// Build window with resize increment hint. Only implemented on X11.
+    ///
+    /// ```no_run
```

Comment:
We could actually allow this example (and others) to run in CI, right?

Reply:
ah, probably, ill see what happens

Reply:
![image](https://user-images.githubusercontent.com/5120858/147334700-a0659ed8-8503-4f34-bac2-e21f56d82bb9.png)
answer seems to be no.
I'll readd the no_run

Reply:
Oh yeah, because of the event loop. But actually I was only thinking of the ones on the `WindowBuilder` (which doesn't require this)

Reply:
Oh right, done!